docs/sources/administration/provisioning/index.md

```diff
@@ -169,56 +169,58 @@ Common settings in the [built-in core data sources]({{< relref "../../datasource
-| postgresVersion            | number  | PostgreSQL                                                       | Postgres version as a number (903/904/905/906/1000) meaning v9.3, v9.4, ..., v10                                                                  |
-| timescaledb                | boolean | PostgreSQL                                                       | Enable usage of TimescaleDB extension                                                                                                             |
-| maxOpenConns               | number  | MySQL, PostgreSQL and MSSQL                                      | Maximum number of open connections to the database (Grafana v5.4+)                                                                                |
-| maxIdleConns               | number  | MySQL, PostgreSQL and MSSQL                                      | Maximum number of connections in the idle connection pool (Grafana v5.4+)                                                                         |
-| connMaxLifetime            | number  | MySQL, PostgreSQL and MSSQL                                      | Maximum amount of time in seconds a connection may be reused (Grafana v5.4+)                                                                      |
-| keepCookies                | array   | _HTTP\*_                                                         | Cookies that needs to be passed along while communicating with data sources                                                                       |
-| prometheusVersion          | string  | Prometheus                                                       | The version of the Prometheus data source, such as `2.37.0`, `2.24.0`                                                                             |
-| prometheusType             | string  | Prometheus                                                       | The type of the Prometheus data sources. such as `Prometheus`, `Cortex`, `Thanos`, `Mimir`                                                        |
-| implementation             | string  | AlertManager                                                     | The implementation of the AlertManager data source, such as `prometheus`, `cortex` or `mimir`                                                     |
-| handleGrafanaManagedAlerts | boolean | AlertManager                                                     | When enabled, Grafana-managed alerts are sent to this Alertmanager                                                                                |
+| Name                          | Type    | Data source                                                      | Description                                                                                                                                           |
```

Comment:
Prettier forces me to update a bunch of whitespace, I apologize for all of the conflicts that this will inevitably introduce